snippets/python/dictionary_using_trie_python.md

```diff
@@ -0,0 +1,151 @@
+# Dictionary using Trie
+
+## Trie
+Trie is an efficient data retrieval structure which builds upon the properties of tree structure. Word search complexitites
+can be brought to an optimal limit which depends on key length.
+
+## Traversing the trie
+Main application of the trie is storing words in the structure. This trie can be traversed by first approaching the root node then traversing along the path on which our 
+letter and the letter within the node's branch matches. You can learn more about inserting and searching, and tries in general [here](https://www.geeksforgeeks.org/trie-insert-and-search/)
```

Comment:
Add this stuff under the Description sub-heading like this:

## Description
### Trie
Trie is an efficient data retrieval structure which builds upon the properties of tree structure. Word search complexitites
can be brought to an optimal limit which depends on key length.

### Traversing the trie
Main application of the trie is storing words in the structure. This trie can be traversed by first approaching the root node then traversing along the path on which our 
letter and the letter within the node's branch matches. You can learn more about inserting and searching, and tries in general [here](https://www.geeksforgeeks.org/trie-insert-and-search/)